README.md

```diff
@@ -89,3 +92,12 @@ for additional information.
   $ mkdir /some/dir/nexus-data && chown -R 200 /some/dir/nexus-data
   $ docker run -d -p 8081:8081 --name nexus -v /some/dir/nexus-data:/nexus-data sonatype/docker-nexus3
   ```
+
+## Getting help
```

Comment:
The "Persistent Data" heading as well as the ToC use title-case, this should probably as well.